### DISCUSSION TITLE
프론트엔드 webpack 대신 vite로 마이그레이션에 대한 논의

### DISCUSSION THREAD

**Discussion by jaryapp** [CMI-OSS/cbnu-alrami#165] [General]
@CMI-OSS/cbnu_alrami_fe 
현재 mobile과 admin에서 `webpack`을 사용하고 있는데 요즘 핫한 `vite`로 전환하면 어떨지에 대해 자유롭게 의견주시면 감사하겠습니다
https://www.youtube.com/watch?v=iX3Nu1FcZKA
https://vitejs-kr.github.io/guide/why.html

**Comment by kingyong9169** [comment]
오 내용 봤는데 신기하게도 되게 빠르네요! 또한 배우는 학생 입장에서 어떤 것이든 다 배우고 싶습니다!ㅎㅎ

**Comment by soonitoon** [comment]
저도 브루니와 같은 의견입니다! 아직 웹팩도 잘 모르긴 하지만 새로운 기술에 도전해보는 건 좋은 것 같아요~ 잠깐 찾아봤는데 마이그레이션 관련해서 참고할 레퍼런스도 많이 나온 것 같습니다

**Comment by nohgijin** [comment]
good~

**Comment by thdwlsgus0** [comment]
vite라는 기술을 들어보긴 했는데 써보는 것은 좋은 것 같아요

그런데 제가 아직 webpack이라는 기술과 vite라는 기술의 명확한 차이를 공부를 못해서 
이 부분에 대한 학습이 필요할 것 같아요
트렌드에 따라가는건 좋지만 기술을 써야하는 이유를 명확히하면 좋을 것 같아서요!

마이그레이션의 경우 일단 이슈로 추가해놓은 다음에 조금씩 변경하는 방법이 좋을 것 같아요 
webpack -> vite는 기능과 ui가 어느 정도 만들어지면 제가 작업할게요!

**Comment by nohgijin** [comment]
이거는 현재 개발에 반영된 디스커션이여서 클로즈할게요!
